tritium-lib/src/main/java/com/palantir/tritium/io/InstrumentedStreams.java

```diff
@@ -34,7 +34,7 @@ private InstrumentedStreams() {}
      * @return instrumented input stream
      */
     public static InputStream input(
-            InputStream in, TaggedMetricRegistry metrics, @Safe @CompileTimeConstant String type) {
+            InputStream in, TaggedMetricRegistry metrics, @Safe @CompileTimeConstant final String type) {
```

Comment:
This shouldn’t be necessary, sometimes these fail when other compilation issues occir

Reply:
hmm, that's odd and removing `final` does seem to pass now that the `NullAway` error in `InvocationEventProxy` that was failing compilation is now fixed.

https://github.com/palantir/tritium/pull/1666/files#diff-89047cd40a88a385fe182637044147a992d694afff22f2c2cdce968c2ab7c77fL84